models.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+from flask import Flask
+
+import sqlite3
+from flask import g
+
+DATABASE = "baza.db"
+
+app = Flask(__name__)
+
+
+# na tyle ile umiałem, nie do końca jeszcze kumam z wyciągnięciem kursora z bloku,

```

Comment:
z wyciąganiem cursora chodzi nam o to żeby tworzyć w każdej metodzie instancję kursora za pomocą wywołania metody `cursor()` obiektu `connection`, do którego masz referencję w samym obiekcie (`self.db`)